charts/redpanda/values.go

```diff
@@ -31,6 +31,8 @@ const (
 	// PostUpgradeContainerName is the user facing name of the post-upgrade
 	// job's container.
 	PostUpgradeContainerName = "post-upgrade"
+
+	certificationMountPoint = "/etc/tls/certs"
```

Comment:
```suggestion
	certificateMountPoint = "/etc/tls/certs"
```

Please add a comment as well :)

Reply:
Done